addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -354,7 +354,9 @@ def _import_retrieve_and_fill_partner_bank_details(self, invoice, bank_details):
         """ Retrieve the bank account, if no matching bank account is found, create it
         """
 
-        bank_details = map(sanitize_account_number, bank_details)
+        # remove the journal from the context, as it may be set to import statements, not to link the partner to it
+        Bank = self.env['res.partner.bank'].with_context(default_journal_id=False)
```

Comment:
I don't understand the comment.
I also don't understand why we set the context here instead of making it more specific for the `create`. Isn't the point of this because we are setting wrong context values in wrong places already?

Also, won't setting the context to `False` actually force the value to be `False` instead of computed or coming from other default parameters? Shouldn't we remove it instead?
By the way, it would be nice if the ORM could have a nice API to remove keys from the context :wink: 
A suggestion: `.with_context(__remove=('default_journal_id',))`

Reply:
I'm cleaning the context because the import implementation may set default values. Multiple functions call this, so it is easier to remove the keys from the context instead of checking each call site.

A remove keyword would be nice.